index.js

```diff
@@ -100,7 +102,10 @@ function useMediaRecorder({
     }
 
     setStatus('acquiring_media');
-
+    if(customMediaStream) {
```

Comment:
What do you think about adding a stricter check here such as `customMediaStream && customMediaStream instanceof MediaStream`?